Bio/SeqIO/UniprotIO.py

```diff
@@ -44,12 +44,6 @@ def UniprotIterator(
     return_raw_comments = True --> comment fields are returned as complete XML to allow further processing
     skip_parsing_errors = True --> if parsing errors are found, skip to next entry
     """
-    if isinstance(alphabet, Alphabet.NucleotideAlphabet):
```

Comment:
I think this should become something like ``"if alphabet and "protein" not in alphabet":`` where the default argument changes to ``alphabet="protein"`` (although we might want to change the argument name here and elsewhere in SeqIO/AlignIO etc)?

Reply:
On second thoughts, we can probably record/verify the molecule type centrally centrally in ``Bio.SeqIO.parse(...)``